### ISSUE TITLE
[Screen Reader][TextField] Narrator doesn't read errorMessage in scan mode when it changes

### ISSUE THREAD

**Issue by emkessle** [microsoft/fluentui#16433]
### Environment Information

- **Package version(s)**: 7.4.0
- **Browser and OS versions**: Edge, Narrator, Windows 10

### Describe the issue:

We are using the TextField and noticed that when the errorMessage changes, the narrator doesn't read the new error text. This is an accessibility bug as the error message needs to be read by the screen reader. We're showing the errorMessage after the user submits one text field value and the check returns an error. We then update the errorMessage to be the new string and re-render. 

### Please provide a reproduction of the issue in a codepen:

https://codepen.io/pen?template=rEjZeV

#### Actual behavior:

Screen reader doesn't announce errorMessage

#### Expected behavior:

Screen reader should say the errorMessage text

**Comment by gouttierre** [comment]
@emkessle - Thanks for reaching out and providing your feedback. 

@khmakoto  - Would you be able to determine if this is a regression or if this behavior is an issue with accessibility?
cc @smhigley

**Comment by Bhavana-Nimma** [comment]
We too face similar kind of issue with TextField. We are using `onBlur` method to show the error message when, the value entered is incorrect and the focus is out. 
But on clicking tab, focus is shifted to next focusable element and the error message which popped up is announced after announcing the element that got focused. Expected is to announce the error message immediately on tabbing out.

**Comment by smhigley** [comment]
@emkessle I think that may be the wrong codepen. That shows a native browser `alert()` (not to be confused with the live region `role="alert"`. The browser `alert()` is a native browser dialog, and not part of Fluent.

Since this bug is about TextField error messages, do you have an isolated repro that shows the bug?
